File: Roadmap/00 - SINTAXIS, VARIABLES, TIPOS DE DATOS Y HOLA MUNDO/php/mensius87.php
<?php

/*
 * EJERCICIO:
 * - Crea un comentario en el código y coloca la URL del sitio web oficial del
 *   lenguaje de programación que has seleccionado.
 * - Representa las diferentes sintaxis que existen de crear comentarios
 *   en el lenguaje (en una línea, varias...).
 * - Crea una variable (y una constante si el lenguaje lo soporta).
 * - Crea variables representando todos los tipos de datos primitivos
 *   del lenguaje (cadenas de texto, enteros, booleanos...).
 * - Imprime por terminal el texto: "¡Hola, [y el nombre de tu lenguaje]!"
 * 
 */

// Existen tres tipos de comentarios:

// Comentario de una sola línea

# Otro comentario de una sola línea con otro formato

/* Comentario multilínea:
    línea
    línea
    línea
*/


/**
 * Comentario de documentación (lleva 2 astericos al principio). Sirve para generar docuemntación
 * automáticamente.
 * Esta es una descripción de la función
 * 
 * Más detalles sobre esta función aquí.
 */

// variables
$variableString = "Hola MoureDev"; // variable String
$variableInteger = 12; // variable entera
$variableFloat = 12.5; // variable decimal
$variableBoolean = true; // variable booleana
$variableArray = array("a", "b","c"); // variable tipo array

// Constantes
const CONSTANTEPI = 3.14; // forma de crear constante (debe de ir en mayúscula)
define("CONSTANTEPI2",3.14); // otra forma de crear constante, también tiene que ir en mayúscula

$nombreLenguage = "PHP";

// Imprimir mensaje con el nombre del lenguaje
echo "¡Hola, " . $nombreLenguage . "!";
?>

File: Roadmap/01 - OPERADORES Y ESTRUCTURAS DE CONTROL/javascript/Roadmap/00 - SINTAXIS, VARIABLES, TIPOS DE DATOS Y HOLA MUNDO/python/mensius87.py
"""
 * EJERCICIO:
 - Crea un comentario en el código y coloca la URL del sitio web oficial del
    lenguaje de programación que has seleccionado.
 - Representa las diferentes sintaxis que existen de crear comentarios
    en el lenguaje (en una línea, varias...).
 - Crea una variable (y una constante si el lenguaje lo soporta).
 - Crea variables representando todos los tipos de datos primitivos
    del lenguaje (cadenas de texto, enteros, booleanos...).
 - Imprime por terminal el texto: "¡Hola, [y el nombre de tu lenguaje]!"
"""


# Sitio oficial de Python: https://www.python.org/

# Comentario de una línea

"""
Este es un comentario multilínea. Aunque Python no tiene una forma de generar este tipo de comentarios, si a una cadena
de texto envuelta entre estas comillas triples no se le asigna una variable, Python la ignorará y servirá de comentario.
"""

variable = "Hola MoureDev"

CONSTANTE = 3,14 # en Python no hay constantes, pero por convención si la variable está en mayúsculas se tomará como tal

entero = 20

flotante = 1.4

texto = "Python"

Booleano = True


print(f"Hola {texto}")
